### PR TITLE
Improvements

### PR DESCRIPTION
**Pull request by petrparolek**
- Makefile: project
- rework htaccess
- fixed PHPStan
- fixed GA

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,28 @@
...
+# DEVELOPMENT ##############################################
+############################################################
 
 qa: cs phpstan
 
 cs:
 	vendor/bin/codesniffer app
 
-cfx:
+csfix:
```
Comment (f3l1x) [inline Makefile:25]: Proc takhle zmena?

File: Makefile [hunk 2]
```diff
@@ -14,12 +31,5 @@ phpstan:
 tests:
 	echo "OK"
 
-tests-coverage:
+coverage:
 	echo "OK"
-
-#####################
```
Comment (f3l1x) [inline Makefile:20]: Tato sekce je potreba.

File: Makefile (hunk 1 above)
Comment (petrparolek) [reply Makefile:25]: Máš to tak všude

File: Makefile (hunk 2 above)
Comment (petrparolek) [reply Makefile:20]: ok

**Pull request merged**